cdap-docs/integrations/source/ambari/index.rst

```diff
@@ -0,0 +1,280 @@
+
+To install the ``cdap-ambari-service`` package, first add the appropriate CDAP repository
+to your system’s package manager by following :ref:`this procedure for installing a Cask
+repository <install-preparing-the-cluster>` on your Ambari server.
+
+The *repository version* must match the *CDAP version* which you’d like installed on your
+cluster. To get the *CDAP 3.0 series,* you would install the *CDAP 3.0 repository.* The
+default is to use CDAP 3.2, which has the widest compatibility with Ambari-supported
+Hadoop distributions.
+
++---------------------------------------------------------+
+| Supported Hortonworks Data Platform (HDP) Distributions |
++----------------+----------------------------------------+
+| CDAP Version   | Hadoop Distributions                   |
++================+========================================+
+| CDAP 3.0.x     | HDP 2.0, HDP 2.1                       |
++----------------+----------------------------------------+
+| CDAP 3.1.x     | HDP 2.0, HDP 2.1, HDP 2.2              |
++----------------+----------------------------------------+
+| CDAP 3.2.x     | HDP 2.0, HDP 2.1, HDP 2.2, HDP 2.3     |
++----------------+----------------------------------------+
+
+**Note:** The CDAP Ambari service has been tested on Ambari Server 2.0 and 2.1, as
+supplied from Hortonworks.
+
+
+.. rubric:: Installing via APT
+
+::
+
+  $ sudo apt-get install -y cdap-ambari-service
+  $ sudo ambari-server restart
+
+.. rubric:: Installing via YUM
+
+::
+
+  $ sudo yum install -y cdap-ambari-service
+  $ sudo ambari-server restart
+
+
+Adding CDAP to Your Cluster
+===========================
+
+.. rubric:: Dependencies
+
+CDAP depends on certain services being present on the cluster. There are **core
+dependencies,** which must be running for CDAP system services to operate correctly, and
+**optional dependencies,** which may be required for certain functionality or program types.
+
+The host running the CDAP Master service must have the HDFS, YARN, and HBase clients
+installed, as CDAP uses the command line clients of these for initialization and their
+connectivity information for external service dependencies. Also, CDAP currently requires
+Internet access on the CDAP service nodes until the issues `CDAP-3957
+<https://issues.cask.co/browse/CDAP-3957>`__ or `AMBARI-13456
+<https://issues.apache.org/jira/browse/AMBARI-13456>`__ are resolved.
+
+.. rubric:: Core Dependencies
+
+- **HDFS:** used as the backing file system for distributed storage
+- **MapReduce2:** used for batch operations in workflows and data exploration
+- **YARN:** used for running system services in containers on cluster NodeManagers
+- **HBase:** used for system runtime storage and queues
+- **ZooKeeper:** used for service discovery and leader election
+
+.. rubric:: Optional Dependencies
+
+- **Hive:** used for data exploration using SQL queries via CDAP Explore system service
+- **Spark:** used for running Spark programs within CDAP applications
```

Comment:
Thanks, and I was wrong. We already merged `3.0`...

Spark isn't available in CDAP 3.0, so we should comment as such.